css/style.css

```diff
@@ -532,9 +547,712 @@
 
   .submit:hover {
     background-color: #fff5e1;
+    color: #f55800;
+  }
+
+  .submit:active {
+    background-color: #f55800;
+  }
+
+  .submit:disabled {
+    background-color: #f4f1f0;
+  }
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
Currently, you have one media query for max-width: 500px and one for min-width: 768px. This means that your webpage _doesn't_ contain any styling between 500px and 768px. Please adjust the first query to max-width: 768px to fix this problem and display your beautiful work on all devices. 😄

Between 500~768px:
![size-500-768](https://user-images.githubusercontent.com/42799579/193004022-596dc5f1-af09-4ca8-b6ae-a16d356afd9b.png)